config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```

Comment:
are we going to store both development and production on amazon S3? If so, do we have differently configured S3 buckets to separate development and production? We'll want to configure two different buckets so that we don't mixup our development images from our production images.  

@ogarocious